cmd/shield/main.go

```diff
@@ -523,7 +530,7 @@ func main() {
 	*/
 
 	c.HelpBreak()
-	c.Dispatch("list retention policies", "List available retention policies",
+	c.Dispatch("list retention policies", "List available retention policies\r\x1b[1A\x1b[1mPOLICIES:\n\x1b[0m",
```

Comment:
ANSI escape sequences are cute and all, but need to take into account whether or not the the current output source is in fact a terminal capable of handling ANSI CSIs.  We used [github.com/jhunt/ansi](https://github.com/jhunt/ansi) for the colorizing of confirmation and error messages in `tui`, so I'd like to see use that here as well.  I don't believe it supports bold _yet_, but that can be fixed.